app/upgrades/v17/upgrades_test.go

```diff
@@ -72,8 +72,6 @@ func dummyTwapRecord(poolId uint64, t time.Time, asset0 string, asset1 string, s
 }
 
 func assertTwapFlipped(suite *UpgradeTestSuite, pre, post types.TwapRecord) {
-	suite.Require().Equal(pre.Asset0Denom, post.Asset1Denom)
-	suite.Require().Equal(pre.Asset1Denom, post.Asset0Denom)
```

Comment:
Can we still have checks for preAsset0Denom and post.Asset0Denom, same for denom1?